### ISSUE TITLE
node.dll error

### ISSUE THREAD

**Issue by johnjjordannet** [tagspaces/tagspaces#627]
when I first try to start the program. How do I fix?

**Comment by uggrock** [comment]
Can you share details about your computer configuration and operating system.

**Comment by johnjjordannet** [comment]
​win 7, 32 bit. We're good now.
I fiddled with it some more, trying to start it from different places that
winzip put it. Was not obvious to me where to start it from. This was
compounded by nothing happening when I tried to access the scanned files.
Turns out it was slow, so I didn't realize it was starting till I tried
other things. We have slow network here (including from scanned file
location to computer with program on it), may be a factor. Anyway I got it
to give me a message that my operating system was not compatible.
Downloaded again, reinstalled again, made sure that ​I was starting it from
the application just downloaded. Waited long enough to let it load. It
worked. Sometimes going slow and careful, being persistent, etc., works.
It has a node.dll file in the download, but I didn't get to figure out what
was going on with that.
Anyway, thanks for your response.
